packages/graphql-serve/bin/gqlserve.ts

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/node
```

Comment:
This is very hacky way of creating commands. Please follow standard we have in the graphback-cli - using cmd wrappers so we can reuse actual command services while keeping commands lightweight in code